src/wallet/wallet.ts

```diff
@@ -94,6 +95,7 @@ class HathorWalletServiceWallet extends EventEmitter implements IHathorWallet {
   private xpub: string | null;
   // Xpriv of the wallet on the account derivation path
   private xpriv: string | null;
+  private authxpriv: string | null;
```

Comment:
Is this the same as the attribute below? `authPrivKey`

Reply:
This property has been removed and we will use the `authPrivKey` instead